src/components/furniture/ShelfHeights.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div>
+    <div class="shelf-height" :style="cssStyle" @click="update = true">
+      <div v-if="update">
+        <input
+          type="number"
+          v-model="newHeight"
+          :placeholder="shelf.height"
+        /><span>cm</span>
+      </div>
+      <p v-else>{{ shelf.height }} cm</p>
+    </div>
+    <div class="button">
+      <button v-if="update" @click="updateFurniture">OK</button>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+export default {
+  props: ['shelf'],
+  data() {
+    return {
+      update: false,
```

Comment:
boolean always are names with isBlue, isLoading, etc. Or can be also hasName etc.